src/templates/pycontw-2020/contents/en/events/passion-fruit.html

```diff
@@ -0,0 +1,88 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}Warm-up Seaasion{% endblock title %}
+
+{% block meta_description %}Overview on how to submit a financial aid application to PyCon Taiwan, and allow us to help you attend the conference.{% endblock meta_description %}
+
+
+{% block content %}
+
+<section class="pill-title alt">
+    <h1>Warm-up Seaasion</h1>
+</section>
+
+<h2>第壹次Python NPF百香果 x 南投.py</h2>
+
+<img src="https://i.imgur.com/FRIbiSE.png">
+<p>Python NPF百香果是今年一個由PyCon Taiwan所籌備的小型線上會議，在疫情肆虐全球的情況下，我們希望藉由這樣的線上活動來拉近彼此間的距離，並利用此本次活動向大家揭露南投.py的神秘面紗！</p>
+
+<ul><li>活動時間:</li></ul>
+<blockquote>
+    <p>2020/05/30 14:00</p>
+</blockquote>
+<ul><li>活動形式:</li></ul>
+<blockquote>
+    <p>Youtube 線上直播</p>
+</blockquote>
+<ul><li>線下聚會:</li></ul>
+<blockquote>
+    <p>768藝術空間</p>
+</blockquote>
+<ul><li>連線資訊:</li></ul>
+<blockquote>
+    <p>請關注 <a href="https://www.facebook.com/pycontw/" target="_blank" rel="noopener">PyCon Taiwan</a>及 <a href="https://www.facebook.com/%E5%8D%97%E6%8A%95python-113209870215893/" target="_blank" rel="noopener">南投.py</a>  Facebook粉專資訊!  Youtube直播連結會於當天發布於FB動態牆。直播結束後，會再將影片上傳至此。</p>
+</blockquote>
+<ul><li>主持人:</li></ul>
+<blockquote>
+    <p>南投.py創辦人 - 林正敏</p>
+</blockquote>
```

Comment:
I think we can do something like:

```html
<ul>
    <li>活動時間<br />
        2020/05/30 14:00
    </li>
    <li style="margin-top: 1.5em">活動形式<br />
        Youtube 線上直播
    </li>
    <li style="margin-top: 1.5em">線下聚會<br />
        768藝術空間
    </li>
    <li style="margin-top: 1.5em">連線資訊<br />
        請關注 <a href="https://www.facebook.com/pycontw/" target="_blank" rel="noopener">PyCon Taiwan</a>及 <a href="https://www.facebook.com/%E5%8D%97%E6%8A%95python-113209870215893/" target="_blank" rel="noopener">南投.py</a>  Facebook粉專資訊!  Youtube直播連結會於當天發布於FB動態牆。直播結束後，會再將影片上傳至此。
    </li>
    <li style="margin-top: 1.5em">主持人<br />
        南投.py創辦人 - 林正敏
    </li>
</ul>
```

It's not ideal, but a start.
And then the second move should be adding a class at [src/static/pycontw-2020/styles/page.scss](src/static/pycontw-2020/styles/page.scss), something like:
```scss
  &.mt {
    margin-top: 1em;
  }
```
inside the `ul { ... > li { ...`